commcare_connect/opportunity/models.py

```diff
@@ -166,6 +166,7 @@ class VisitValidationStatus(models.TextChoices):
     pending = "pending", gettext("Pending")
     approved = "approved", gettext("Approved")
     rejected = "rejected", gettext("Rejected")
+    extra = "extra", "extra"
```

Comment:
The second element should use `gettext` for translation like the others and follow the same capitalization convention